lib/classes/calculator_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+class Calculator
+  
+  def add(list_of_numbers)
+    list_of_numbers.sum
+  end
+  
+  def max(list_of_numbers)
+    list_of_numbers.max
+  end
+  
+  def min(list_of_numbers)
+    list_of_numbers.min
+  end
+  
+  def odd(list_of_numbers)
+    list_of_numbers.sum.odd?
+  end
+  
+  def even(list_of_numbers)
+    list_of_numbers.sum.even?
+  end
+  
+  def self.hello
+   "Hello, I am your Calculator :)"
+  end
+end
+
+describe Calculator do 
+  let(:calculator) {Calculator.new}
```

Comment:
please use subject 

subject { Calculator.new }

then inside it blocks you can use 
expect(subject.add([2,4,5,1])).to eq(12)